app/catalog-tab/launch/route.js

```diff
@@ -122,6 +122,10 @@ export default Route.extend({
           ( { namespace, newAppName } = this.newNamespace(existingNamespace, namespaceName));
         }
 
+        if ( params.istio === 'true' ) {
```

Comment:
If possible we should use the primitive bool type not a string rep in queryPrams. I understand the limitations of when we're using this param else where but since we control the queryParams we should use the correct type when possible.